### PR TITLE
Views

### PR DESCRIPTION
**Pull request by Mesi21**
- add the bootstrap gem to the project
- Fill the methods index and show for users_controller
- Fill the methods index and show for posts_controller
- Add view files for users and posts

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,6 @@
-<h1>This is the index page where all users of the app will arrive</h1>
+<section class='all-users'>
+  <h1 class='shadow'>ALL OUR BLOGERS</h1>
+  <% @users.each do |user| %>
+    <%= render partial: 'user_box', locals: {user:user} %>
```
Comment (Whoistolu) [inline app/views/users/index.html.erb:4]: - [ ] [OPTIONAL] You did a great job rendering a partial here 🥇

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,33 @@
...
+      <li class="list-group-item"><%= p.text.truncate(90) %></li>
+      <li class="list-group-item d-flex flex-row" id="lks-posts">
+        <%= link_to 'View post', user_post_path(@user, p), class: 'btn btn-view' %>
+        <p>Comments: <%= p.comments_counter %></p>
+        <p>Likes: <%= p.likes_counter %></p>
+      </li>
+      <li>
+        <% p.last_five_comments.each do |c| %>
+            <p><strong><%= c.user.name %></strong>: <%= c.text %></p>
+        <% end %>
```
Comment (Whoistolu) [inline app/views/posts/index.html.erb:26]: - [ ] [OPTIONAL] Here, I changed the variable name from c to comment. This makes the code more self-explanatory, and someone reading the code can immediately understand that the variable represents an individual comment. Descriptive variable names contribute to code clarity, especially when revisiting the code or when others are reading it.

``` ruby
<% p.last_five_comments.each do |comment| %>
  <p><strong><%= comment.user.name %></strong>: <%= comment.text %></p>
<% end %>
```

**Review by Whoistolu — APPROVED** [review]
Great job 🥇

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1 +1,33 @@
...
+      </li>
+      <% i += 1 %>
+      <li class="list-group-item"><%= p.text.truncate(90) %></li>
+      <li class="list-group-item d-flex flex-row" id="lks-posts">
+        <%= link_to 'View post', user_post_path(@user, p), class: 'btn btn-view' %>
+        <p>Comments: <%= p.comments_counter %></p>
+        <p>Likes: <%= p.likes_counter %></p>
+      </li>
+      <li>
+        <% p.last_five_comments.each do |c| %>
```
Comment (Whoistolu) [inline app/views/posts/index.html.erb:24]: - [x] [OPTIONAL] Here, I changed the variable name from c to comment. This makes the code more self-explanatory, and someone reading the code can immediately understand that the variable represents an individual comment. Descriptive variable names contribute to code clarity, especially when revisiting the code or when others are reading it.

``` ruby
<% p.last_five_comments.each do |comment| %>
  <p><strong><%= comment.user.name %></strong>: <%= comment.text %></p>
<% end %>
```

File: app/views/users/index.html.erb (hunk 1 above)
Comment (Whoistolu) [inline app/views/users/index.html.erb:4]: - [x] [OPTIONAL] You did a great job rendering a partial here and in other parts of the code-base 🥇

**Review by Whoistolu — APPROVED** [review]
### Hi @Mesi21 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

### Highlight
🔆 Users controller `#index` action renders all users to it's corresponding Views index page
🔆 Clicking on each user takes you to their show page
🔆 Uses `#last_three_posts` method to render the user's 3 most recent post.
🔆 **See all Post** links opens user's post index

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (Mesi21) [reply app/views/users/index.html.erb:4]: Thank you!

File: app/views/posts/index.html.erb (hunk 3 above)
Comment (Mesi21) [reply app/views/posts/index.html.erb:24]: Thanks for pointing this out. I agree with you, so I will change it later while working on adding forms.

**Pull request merged**